ssas/README.md

```diff
@@ -4,76 +4,66 @@ The SSAS can be run as a standalone web service or embedded as a library.
-| OKTA_MFA_EMAIL       | No       | The email address (Okta account identifier) for the account to test in the Okta sandbox. Required only if running the live Okta MFA tests. |
-| OKTA_MFA_USER_ID     | No       | The user ID for the account to test in the Okta sandbox. Required only if running the live Okta MFA tests. |
-| OKTA_MFA_USER_PASSWORD| No      | The password for the account to test in the Okta sandbox. Required only if running the live Okta MFA tests. |
-| OKTA_MFA_SMS_FACTOR_ID | No     | The SMS MFA factor ID enrolled for the account to test in the Okta sandbox. Required only if running the live Okta MFA tests. |
-| SSAS_READ_TIMEOUT    | No       | Sets the read timeout on server requests |
-| SSAS_WRITE_TIMEOUT   | No       | Sets the write timeout on server responses |
-| SSAS_IDLE_TIMEOUT    | No       | Sets the idle timeout on |
-| SSAS_ADMIN_SIGNING_KEY_PATH  | Yes | Provides the location of the admin server signing key |
-| SSAS_PUBLIC_SIGNING_KEY_PATH | Yes | Provides the location of the public server signing key |
-| SSAS_TOKEN_BLACKLIST_CACHE_CLEANUP_MINUTES | No | Tunes the frequency that expired entries are cleared from the token blacklist cache.  Defaults to 15 minutes. |
-| SSAS_TOKEN_BLACKLIST_CACHE_TIMEOUT_MINUTES | No | Sets the lifetime of token blacklist cache entries.  Defaults to 24 hours.
-| SSAS_TOKEN_BLACKLIST_CACHE_REFRESH_MINUTES | No | Configures the number of minutes between times the token blacklist cache is refreshed from the database. |
+Required values must be present in the docker-compose.*.yml files. Some values are primarily for the use of the ACO API, and are only used by SSAS for testing purposes. Some values are only used by the ACO API; they are listed for reference.
+
+Very long keys have been split across two rows for formatting purposes.
+
+|  Key                 | Required | SSAS | BCDA | Purpose |
+| -------------------- |:--------:|:----:|:---:| ------- |
+| BCDA_AUTH_PROVIDER   | Yes      |   | X | Tells ACO API which auth provider to use |
+| BCDA_CA_FILE         | Yes      |   | X | Tells ACO API the certificate file with which to validate its TLS connection to SSAS. When setting vars for AWS envs, you must include a var for the key material  |
+| BCDA_SSAS_CLIENT_ID  | Yes      |   | X | Tells ACO API the client_id to use with the SSAS REST API. |
+| BCDA_SSAS_SECRET     | Yes      |   | X | Tells ACO API the secret to use with the SSAS REST API. |
+| SSAS_USE_TLS         | Yes      |   | X | Should be renamed to BCDA_SSAS_USE_TLS |
+| SSAS_URL             | Yes      |   | X | The url of the SSAS admin server. Should be renamed to BCDA_SSAS_URL |
+| SSAS_PUBLIC_URL      | Yes      |   | X | The url of the SSAS public server (auth endpoints). Should be renamed to BCDA_SSAS_PUBLIC_PUBLIC |
```

Comment:
When should these vars be renamed? When used for BCDA?

Is `BCDA_SSAS_PUBLIC_PUBLIC` correct?